src/App.tsx

```diff
@@ -160,6 +162,10 @@ function App() {
   }
 
   const size = getSize(board.numCols, board.numRows);
+  const boomshake = {
+    shake: {x: 10, transition: {type: 'spring', stiffness: 2000}},
```

Comment:
I experimented with some keyframes and like these values to really make it chaotic for a couple of seconds

```suggestion
    shake: {x: [0, 20, -20, 10, -10, 0], y: [0, 20, -20, -10, 10, 0], transition: {type: 'spring', stiffness: 2000}},
```

Reply:
I just tried out the new values. Your solution works way better. It even allows for bypassing the 'slide back to normal' animation that I couldn't figure out how to remove.

Committing now.